config/initializers/spree.rb

```diff
@@ -86,14 +86,14 @@
   # Please note that you need to use the solidus_stripe gem to have
   # Stripe working: https://github.com/solidusio-contrib/solidus_stripe
   #
-  # config.static_model_preferences.add(
-  #   Spree::PaymentMethod::StripeCreditCard,
-  #   'stripe_env_credentials',
-  #   secret_key: ENV['STRIPE_SECRET_KEY'],
-  #   publishable_key: ENV['STRIPE_PUBLISHABLE_KEY'],
-  #   server: Rails.env.production? ? 'production' : 'test',
-  #   test_mode: !Rails.env.production?
-  # )
+  config.static_model_preferences.add(
+    Spree::PaymentMethod::StripeCreditCard,
+    'stripe_env_credentials',
+    secret_key: ENV['STRIPE_SECRET_KEY'],
+    publishable_key: ENV['STRIPE_PUBLISHABLE_KEY'],
+    server: Rails.env.production? ? 'production' : 'test',
```

Comment:
We should see if the Heroku app also works with the testing credentials with this condition. If not, we may need just to set `test` always. And the same with the next line.